pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -752,3 +738,17 @@ func makeGvk(kind string) schema.GroupVersionKind {
 		Kind:    kind,
 	}
 }
+
+func shouldGenerateVAP(ct *templates.ConstraintTemplate, generateVAPDefault bool) (bool, error) {
```

Comment:
do you want to use hasvapcel() here as well to be consistent?

Reply:
`GetSourceFromTemplate` checks for the `CEL` engine as well.